amd_openvx_extensions/amd_media/CMakeLists.txt

```diff
@@ -88,11 +88,25 @@ elseif (GPU_SUPPORT AND "${BACKEND}" STREQUAL "HIP")
 endif()
 
 install(TARGETS ${PROJECT_NAME} DESTINATION lib)
-install(FILES include/vx_amd_media.h DESTINATION include)
+install(FILES include/vx_amd_media.h DESTINATION include/mivisionx)
 
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
     set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /MT")
     set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} /MTd")
 else()
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-deprecated-declarations -msse4.2 -std=c++11")
 endif()
+
+if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY)
+  #include(ROCMHeaderWrapper)
+  # Copy Header files to build directory
+  configure_file("include/vx_amd_media.h" "${PROJECT_BINARY_DIR}/include/mivisionx/vx_amd_media.h" COPYONLY)
```

Comment:
change "${PROJECT_BINARY_DIR}/include/mivisionx/vx_amd_media.h" to
"${PROJECT_BINARY_DIR}/include/vx_amd_media.h"